### PR TITLE
update support page. 

### LINKED ISSUE #887: Update /support
Smaller updates based on the [internal discussion](https://gitpod.slack.com/archives/C01KGM9ETU6/p1629984090014500?thread_ts=1629980868.012200&cid=C01KGM9ETU6)

- [x] see the updated mockup (smaller search bar and CTAs, 2 CTAs instead of 1, updated resource bubbles, remove guide bubble, reusing blog module from /blogs (please use the existing content from /blog)
- [x] add a new subject (report a bug) to /contact which is active when clicking on "contact support"

Could you look into this @nisarhassan12 ?

mockup: https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=1247%3A194
Support (2)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #887 

This is how it looks:

image

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/897"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/897"></a>

### CODE REVIEW

File: src/contents/support.ts [hunk 1]
```diff
@@ -13,10 +13,25 @@ export const resources: Resource[] = [
...
+	},
+	{
+		title: 'Forum',
 		text: 'If you are looking for help with common requests, connect with our community on Discourse.',
 		href: "https://community.gitpod.io/"
 	},
+	{
+		title: 'Community Chat',
+		text: 'Would you like to take part in live coding sessions and connect with the community ? Then join us on Discord.',
+		href: "/changelog"
```
Comment (ChristinFrohne) [inline src/contents/support.ts:28]: This should link to the Discord channel
https://www.gitpod.io/chat

**Review by ChristinFrohne — APPROVED** [review]
See two minor comments inline. Otherwise, it looks good to me. Thanks :)

**Comment by nisarhassan12** [comment]
> See two minor comments inline.

Thanks. @ChristinFrohne I only see one minor comment inline. Looks like somehow you missed the second one.

**Comment by ChristinFrohne** [comment]
Ups sorry @nisarhassan12 
I was asking to replace the blog post "dev env as code" with "continuous dev environments" (https://www.gitpod.io/blog/continuous-dev-environment-in-devops). I must have forgotten to press the submit button 🤦‍♀️

**Pull request merged**